abdulfataiaka/pluralsight-admin
# #161345849 Implement delete course feature

#### What does this PR do
- Adds implementations for deleting courses

#### Description of work done
- Add delete button to courses table
- Add action and reducer case for course delete

#### How should it be manually tested
- Clone the repository `git clone https://github.com/abdulfataiaka/pluralsight-admin.git`
- Checkout to this branch
- Ensure to see delete button for each course on course list
- Ensure you can delete a course

#### Relevant pivotal tracker story
[161345849](https://www.pivotaltracker.com/story/show/161345849)

#### Screenshots
<img width="1440" alt="screen shot 2018-10-19 at 3 57 37 pm" src="https://user-images.githubusercontent.com/32123313/47226156-bb522980-d3b7-11e8-800f-ef27316caf1a.png">
